app/definitions/ISubscription.ts

```diff
@@ -79,6 +79,7 @@ export interface ISubscription {
 	avatarETag?: string;
 	teamId?: string;
 	teamMain?: boolean;
+	sms?: boolean;
```

Comment:
Isn't sms a dynamic column on Omnichannel?
If so, we shouldn't add it here.
Same for the code, as we shouldn't count on having it.